src/bonsai/websocket/markets.ts

```
@@ -110,7 +110,54 @@ export function setUpMarkets(store: RootStore) {
           return;
         }
         AppStartupTimer.timeIfFirst('loadedMarkets');
-        setMarketsIfEmpty(loadableLoaded(allMarkets.markets));
+
+        // Transform the raw market data into the final PerpetualMarketSummary format
+        const transformedMarkets = Object.entries(allMarkets.markets).reduce((acc, [marketId, market]) => {
```

ESLint

[error] Insert `⏎··········`

(prettier/prettier)